analyzing-shakespeare/lib/macbeth/base_request.rb

```diff
@@ -0,0 +1,9 @@
+require 'net/http'
+
+class BaseRequest
+  MACBETH_URL = URI('http://www.ibiblio.org/xml/examples/shakespeare/macbeth.xml')
+
+  def macbeth_xml
```

Comment:
i'w rather call this something like
fetch_xml

Reply:
If you have some atomic operation like this(one thing you do here is getting XML)
so i good place to use here singleton an rewrite it as class method